covid-dashboard/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "covid-dashboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "lint": "eslint ./src",
+    "lint:fix": "eslint ./src --fix",
+    "watch": "webpack --mode development --watch",
+    "dev-server": "webpack-dev-server --mode development --open"
```

Comment:
Can you run: npm run dev-server? Does dev-server runs successfully? We use Webpack 5, so "dev-server" comand should be: "webpack serve --mode development". And webpack.config.js file should includes field "devServer: { open: true,   port: 4200 }" instead "devServer: { port: 4200 }".